contracts/mainnet/connectors/morpho-blue/libraries/SafeTransferLib.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {IERC20} from "../interfaces/IERC20.sol";
+
+import {ErrorsLib} from "../libraries/ErrorsLib.sol";
+
+interface IERC20Internal {
+    function transfer(address to, uint256 value) external returns (bool);
+    function transferFrom(address from, address to, uint256 value) external returns (bool);
+}
+
+/// @title SafeTransferLib
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice Library to manage transfers of tokens, even if calls to the transfer or transferFrom functions are not
+/// returning a boolean.
+/// @dev It is the responsibility of the market creator to make sure that the address of the token has non-zero code.
```

Comment:
I'm warning you there's a pending change in this library, merged in branch `morpho-blue/post-cantina`: https://github.com/morpho-org/morpho-blue/pull/629

While the content of the library you copied was based on `morpho-blue/main`

This is a critical change for the protocol, it may also be one for yours
But IMO, if you don't use this library, you should even consider not having it copied! Same for `EventsLib` or `ErrorsLib` for example!

Reply:
Updated the libraries